datastore/db/metadata/inspection/__init__.py

```diff
@@ -347,43 +361,49 @@ def build_inspection_export(cursor, inspection_id, label_info_id) -> str:
                 SubLabel(**sub_label_json[key])
 
         inspection_json.update(sub_label_json)
-        # Get the ingredients
-        ingredients_json = ingredient.get_ingredient_json(cursor, label_info_id)
+        # # Get the ingredients
+        # ingredients_json = ingredient.get_ingredient_json(cursor, label_info_id)
 
-        ValuesObjects(**ingredients_json)
+        # ValuesObjects(**ingredients_json)
 
-        inspection_json.update(ingredients_json)
+        # inspection_json.update(ingredients_json)
 
-        # Get the nutrients
-        nutrients_json = nutrients.get_micronutrient_json(cursor, label_info_id)
+        # # Get the nutrients
+        # nutrients_json = nutrients.get_micronutrient_json(cursor, label_info_id)
 
-        ValuesObjects(**nutrients_json)
+        # ValuesObjects(**nutrients_json)
 
-        inspection_json.update(nutrients_json)
+        # inspection_json.update(nutrients_json)
 
         # Get the guaranteed analysis
         guaranteed_analysis_json = nutrients.get_guaranteed_analysis_json(
             cursor, label_info_id
         )
+        # print(guaranteed_analysis_json)
         if guaranteed_analysis_json.get("guaranteed_analysis") is None:
-            guaranteed_analysis_json["guaranteed_analysis"] = []
-        for i in range(len(guaranteed_analysis_json.get("guaranteed_analysis"))):
-            Value(**(guaranteed_analysis_json.get("guaranteed_analysis")[i]))
+            guaranteed_analysis_json["guaranteed_analysis"] = GuaranteedAnalysis().model_dump()
+        else:
+            GuaranteedAnalysis(**guaranteed_analysis_json)
 
         inspection_json.update(guaranteed_analysis_json)
 
-        # Get the specifications
-        specifications_json = specification.get_specification_json(
-            cursor, label_info_id
-        )
+        # # Get the specifications
+        # specifications_json = specification.get_specification_json(
+        #     cursor, label_info_id
+        # )
+
+        # Specifications(**(specifications_json.get("specifications")))
 
-        Specifications(**(specifications_json.get("specifications")))
+        # inspection_json.update(specifications_json)è
 
-        inspection_json.update(specifications_json)
+        # Get inspection comment
+        inspection_comment = inspection.get_inspection(cursor,inspection_id)[8]
```

Comment:
This is the function to get all the fields from an inspection. As for the magic number you only need to read the documentation of `get_inspection` to understand what it means.

I could create a function to only retrieve the comment, but it seems trivial and brings no value to the project itself. 

I'd rather keep it as it currently is since all the tests also uses `get_inspection` and refer to the right column to verify the right object.

Reply:
I understand. But this is a code smell and pretty easy to avoid. An alternative to using a model is to simply declare a constant variable with an indicative name. Less attractive than a model tbh, because of how many constants would be needed if we wanted to extract the other values.

The documentation doesn't seem to indicate what 8 means.

Reply:
Wdym
> An alternative to using a model is to simply declare a constant with an indicative name

I feel like returning a 2D array is standard while querying database rows no?

>he documentation doesn't seem to indicate what 8 means.

I can add a line to specify what the inspection is as follows:
```python
def get_inspection(cursor, inspection_id):
    """
    This function gets the inspection from the database.

    Parameters:
    - cursor (cursor): The cursor of the database.
    - inspection_id (str): The UUID of the inspection.

    Returns:
    - The inspection.
    [ verified, upload_date, updated_at, inspector_id, label_info_id, sample_id, picture_set_id, fertilizer_id, user_comment]
    """

    try:
        query = """
            SELECT 
                verified,
                upload_date,
                updated_at,
                inspector_id,
                label_info_id,
                sample_id,
                picture_set_id,
                fertilizer_id,
                user_comment
            FROM 
                inspection
            WHERE 
                id = %s
            """
...
```

Other than that, I do not know what else I can do to make it more obvious that the 8th index is the user_comment. I could create 8 constant in the module to make it more easy to manage if you'd like. However I've always used array indexes to select a column with a Row in an Array.

Reply:
Suppose, get_inspection was used in thousand places throughout the code or even accross repos then we change get_inspection is a way that user_comment changes position to 7. This would mean changing 8 to 7 everywhere. With a constant, you would change it only once. With a model, you would change the parsing code only.

But this is not a huge problem. Resolving this.